app/controllers/home_controller.rb

```diff
@@ -0,0 +1,3 @@
+class HomeController < ApplicationController
+
+end
```

Comment:
I'd add an empty `index` method here, just so it's clear that there's at least one active handler for this controller:
```
def index
end
```